### ISSUE TITLE
Markers in logarithmic Y-axis shows wrong value

### ISSUE THREAD

**Issue by robint91** [jetperch/pyjoulescope_ui#104]
So when I put the Y-axis in log mode, the value of the markers are "wrong". I'm using version 0.9.2.

If you see the bottom green line, it shows -5.14A which is wrong but If I do 10^(-5.14) = 6 uA I get the correct result.

![image](https://user-images.githubusercontent.com/853861/97370481-59ceca80-18af-11eb-91f6-59c5c9c9f875.png)

**Comment by mliberty1** [comment]
Thanks for the bug report @robint91.  I can easily duplicate this issue, and it looks like we neglected logarithmic mode when we added the y-axis markers!  As you observed, the y-axis value is displayed as the power of 10, not the normal decimal value, when in logarithmic scale mode.  I also noticed that the marker value is not correctly maintained when switching between linear and logarithmic modes.

We will fix this in the next release.
